binding/nodejs/README.md

```diff
@@ -0,0 +1,68 @@
+# Cobra Binding for Node.js
+
+## Cobra Voice Activity Detection Engine
+
+Made in Vancouver, Canada by [Picovoice](https://picovoice.ai)
+
+Cobra is an on-device streaming voice activity detection engine. Cobra is:
+
+- Private; All voice processing runs locally.
+- Accurate [[1]](https://picovoice.ai/docs/benchmark/vad/#results)
+- Cross-Platform:
+    - Linux (x86_64), macOS (x86_64, arm64), and Windows (x86_64)
+    - Android and iOS
+    - Chrome, Safari, Firefox, and Edge
+    - Raspberry Pi (2, 3, 4, 5) NVIDIA Jetson Nano, and BeagleBone
+
+## Compatibility
+
+- Node.js 16+
+- Runs on Linux (x86_64), macOS (x86_64, arm64), Windows (x86_64), Raspberry Pi (2, 3, 4, 5), NVIDIA Jetson Nano, and BeagleBone.
+
+## Installation
+
+```console
+npm install @picovoice/cobra-node
+```
+
+## AccessKey
+
+Cobra requires a valid Picovoice `AccessKey` at initialization. `AccessKey` acts as your credentials when using Cobra SDKs.
+You can get your `AccessKey` for free. Make sure to keep your `AccessKey` secret.
+Signup or Login to [Picovoice Console](https://console.picovoice.ai/) to get your `AccessKey`.
+
+### Usage
+
+Create an instance of the engine and detect voice activity in the audio:
+
+Replace `${AccessKey}` with your AccessKey obtained from [Picovoice Console](https://console.picovoice.ai/). `handle` is
+an instance of Cobra.
```

Comment:
It looks like you meant to add a code example here for the init, but combined it with the one below. Move the init up to here.